services/scripts/testy-snapshot.sh

```diff
@@ -211,6 +217,14 @@ create_volume_from_snapshot() {
         return 1
     fi
 
+    # Update volume name.
+    local _ltname
+    _ltname=$(aws ec2 describe-instances --instance-ids "$_instance_id" \
```

Comment:
The variable `_instance_id` is not defined here

Reply:
Fixed in [6afcb40](https://github.com/wiredtiger/testy/pull/50/commits/6afcb408ff6a150f84ade3514b4c2e12302ecbb4).